frontend/src/Settings/Tags/Tags.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import FieldSet from 'Components/FieldSet';
+import PageSectionContent from 'Components/Page/PageSectionContent';
+import TagConnector from './TagConnector';
+import styles from './Tags.css';
+
+function Tags(props) {
+  const {
+    items,
+    ...otherProps
+  } = props;
+
+  if (!items.length) {
+    return (
+      <div>No tags have been added yet</div>
```

Comment:
Should we link to something or provide directions here?

Reply:
I added a wiki page: https://github.com/lidarr/Lidarr/wiki/Tags. Pretty basic, but should give us something to link to.